src/pybind/mgr/cephadm/module.py

```diff
@@ -2148,6 +2148,7 @@ def _schedule_daemon_action(self, daemon_name: str, action: str) -> str:
             raise OrchestratorError(
                 f'Unable to schedule redeploy for {daemon_name}: No standby MGRs')
         self.cache.schedule_daemon_action(dd.hostname, dd.name(), action)
+        self.cache.save_host(dd.hostname)
```

Comment:
assert `save_host` was called during the unit test?

```patch
diff --git a/src/pybind/mgr/cephadm/tests/test_cephadm.py b/src/pybind/mgr/cephadm/tests/test_cephadm.py
index 76a1073fc4a..1b2ba7ec494 100644
--- a/src/pybind/mgr/cephadm/tests/test_cephadm.py
+++ b/src/pybind/mgr/cephadm/tests/test_cephadm.py
@@ -404,13 +404,15 @@ class TestCephadm(object):
         ]
     )
     @mock.patch("cephadm.serve.CephadmServe._run_cephadm", _run_cephadm('{}'))
-    def test_daemon_check(self, cephadm_module: CephadmOrchestrator, action):
+    @mock.patch("cephadm.module.HostCache.save_host")
+    def test_daemon_check(self, save_host, cephadm_module: CephadmOrchestrator, action):
         with with_host(cephadm_module, 'test'):
             with with_service(cephadm_module, ServiceSpec(service_type='grafana'), CephadmOrchestrator.apply_grafana, 'test') as d_names:
                 [daemon_name] = d_names
 
                 cephadm_module._schedule_daemon_action(daemon_name, action)
 
+                save_host.assert_called_with('test')
                 assert cephadm_module.cache.get_scheduled_daemon_action(
                     'test', daemon_name) == action
```

Reply:
done